### DISCUSSION TITLE
Brave and localhost (FYI)

### DISCUSSION THREAD

**Discussion by fabswt** [maizzle/framework#1014] [General]
- Maizzle Version: 4.4.5
- Node.js Version: v19.8.1

FYI I struggled accessing these URLs via localhost in **Brave**:

```
       Local: http://localhost:3002
          UI: http://localhost:3003
```

That's because Brave keeps redirecting http://localhost to https://localhost which then results in this error:

```
This site can’t provide a secure connection
localhost sent an invalid response.
ERR_SSL_PROTOCOL_ERROR
```

Meanwhile, things work fine in Chrome and Safari.

FYI here's what's fixed it:

- In brave://flags/ enabled *Allow invalid certificates for resources loaded from localhost.*
- In brave://net-internals/#hsts used *Delete domain security policies* to clear localhost

This is not an issue with Maizzle itself, but I wasted so much time with it I figured I'd document it somewhere.

(I'll probably just stop using Brave as it seems to more often waste rather than save it.)

**Comment by cossssmin** [comment]
You can use the `https` option for browsersync:

https://browsersync.io/docs/options#option-https

https://maizzle.com/docs/configuration/browsersync
